### ISSUE TITLE
Had to update libs after new account

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-dev#5483]
Should libs not be up to date after account creation?

**Comment by bradmac** [comment]
No, libs should be whatever is specified in the apps dependencies.

We wont always necessarily update app dependencies to the latest version of libs, since we should generally test whenever we change them to ensure compatbiility.

So thats not an issue generally.

But if there is some specific app which needs an updated lib version and has not been updated then that is an issue.
